Makefile

```diff
@@ -16,57 +16,63 @@ wait_for_artifacts:
+	docker run --rm -v $(shell pwd)/test-sql:/flyway/sql $(EDITION)/flyway:$(VERSION) -licenseKey="$(KEY)" -url=jdbc:h2:mem:test migrate
+	docker run --rm -v $(shell pwd)/test-sql:/flyway/sql $(EDITION)/flyway:$(VERSION) -licenseKey="$(KEY)" -url=jdbc:h2:mem:test clean -cleanDisabled=false
+	docker run --rm -v $(shell pwd)/test-sql:/flyway/sql $(EDITION)/flyway:$(VERSION)-alpine -licenseKey="$(KEY)" -url=jdbc:h2:mem:test info
+	docker run --rm -v $(shell pwd)/test-sql:/flyway/sql $(EDITION)/flyway:$(VERSION)-alpine -licenseKey="$(KEY)" -url=jdbc:sqlite:test check -changes -code -check.buildUrl=jdbc:sqlite:temp -check.reportFilename=report
+	docker run --rm -v $(shell pwd)/test-sql:/flyway/sql $(EDITION)/flyway:$(VERSION)-alpine -licenseKey="$(KEY)" -url=jdbc:h2:mem:test migrate
+	docker run --rm -v $(shell pwd)/test-sql:/flyway/sql $(EDITION)/flyway:$(VERSION)-alpine -licenseKey="$(KEY)" -url=jdbc:h2:mem:test clean -cleanDisabled=false
+	docker run --rm -v $(shell pwd)/test-sql:/flyway/sql $(EDITION)/flyway:$(VERSION)-azure flyway -licenseKey="$(KEY)" -url=jdbc:h2:mem:test info
+	docker run --rm -v $(shell pwd)/test-sql:/flyway/sql $(EDITION)/flyway:$(VERSION)-azure flyway -licenseKey="$(KEY)" -url=jdbc:sqlite:test check -changes -code -check.buildUrl=jdbc:sqlite:temp -check.reportFilename=report
+	docker run --rm -v $(shell pwd)/test-sql:/flyway/sql $(EDITION)/flyway:$(VERSION)-azure flyway -licenseKey="$(KEY)" -url=jdbc:h2:mem:test migrate
+	docker run --rm -v $(shell pwd)/test-sql:/flyway/sql $(EDITION)/flyway:$(VERSION)-azure flyway -licenseKey="$(KEY)" -url=jdbc:h2:mem:test clean -cleanDisabled=false
 
 release: URL = https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/
+release: EDITION = flyway
+release: PLATFORM = linux/arm/v7,linux/arm64/v8,linux/amd64
 release:
-	docker buildx build --push --platform linux/arm/v7,linux/arm64/v8,linux/amd64 --build-arg FLYWAY_VERSION=$(VERSION) --build-arg FLYWAY_ARTIFACT_URL=$(URL) \
-	-t flyway/flyway:latest \
-	-t flyway/flyway:$(VERSION) \
-	-t flyway/flyway:$(subst $S,.,$(wordlist 1,2,$(subst .,$S,$(subst -,$S,$(VERSION)))))$(wordlist 2,2,$(subst -,$S-,$(VERSION))) \
-	-t flyway/flyway:$(subst $S,.,$(wordlist 1,1,$(subst .,$S,$(subst -,$S,$(VERSION)))))$(wordlist 2,2,$(subst -,$S-,$(VERSION))) .
-	docker push -a flyway/flyway
+	docker buildx build --target $(EDITION) --push --platform $(PLATFORM) --build-arg FLYWAY_VERSION=$(VERSION) --build-arg FLYWAY_ARTIFACT_URL=$(URL) \
+	-t $(EDITION)/flyway:latest \
+	-t $(EDITION)/flyway:$(VERSION) \
+	-t $(EDITION)/flyway:$(subst $S,.,$(wordlist 1,2,$(subst .,$S,$(subst -,$S,$(VERSION)))))$(wordlist 2,2,$(subst -,$S-,$(VERSION))) \
+	-t $(EDITION)/flyway:$(subst $S,.,$(wordlist 1,1,$(subst .,$S,$(subst -,$S,$(VERSION)))))$(wordlist 2,2,$(subst -,$S-,$(VERSION))) .
+	docker push -a $(EDITION)/flyway
 	git commit --allow-empty -a -m 'Update to $(VERSION)'
-	git tag v$(VERSION) --force
+	git tag v$(VERSION)
```

Comment:
We should put the `--force` back for now since this will still be called twice, once for each edition